hbase-common/src/main/java/org/apache/hadoop/hbase/io/TimeRange.java

```diff
@@ -28,9 +28,6 @@
  * <p>
  * Evaluated according to minStamp &lt;= timestamp &lt; maxStamp
  * or [minStamp,maxStamp) in interval notation.
- * <p>
- * Can be returned and read by clients.  Should not be directly created by clients.
```

Comment:
I think that this comment is still valid until the deprecated constructors are removed.

Reply:
I've added this in the committed patch.